package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
```

Comment:
Think we can just reset this file for now, don't need the version changes especially.